Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -78,11 +79,7 @@ - (instancetype)initWithCollectionView:(UICollectionView *)collectionView
 
 - (NSMutableArray *)collapsedForms
 {
-    if (_collapsedForms) return _collapsedForms;
-
-    _collapsedForms = [NSMutableArray new];
-
-    return _collapsedForms;
+    return _collapsedForms = (_collapsedForms) ?: [NSMutableArray new];
```

Comment:
I see the intention, but I'm a bit unsure about the "true" outcome, in other words: is `(_collapsedForms) ?: [NSMutableArray new]` equivalent to `(_collapsedForms) ? (_collapsedForms) : [NSMutableArray new]` or `(_collapsedForms) ? nil : [NSMutableArray new]`